main.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/rdegges/go-ipify"
 )
 
-const (
+var (
```

Comment:
Why the change?

Reply:
Because of `-s -w -X main.Version={{.Version}}` in .goreleaser.yml. Essentially this sets `Version` to either the tag value for a release or the commit sha. So (1) you do not have to set it on your own when releasing, (2) `trawl -v` clearly communicates whether the binary is a snapshot, a released version or built using go install.

To accomplish (2) I will push a commit changing the default to `manual`